lib/tutorials/cookies.md

```diff
@@ -25,7 +25,7 @@ This configuration will make it so the cookie named `data` has a session time-li
 
 In addition to this, you may pass two parameters to the `config` when adding a route:
 
-```javascript
+```json5
```

Comment:
Object literals alone aren't valid javascript– try copying and pasting this code block into a node script / browser console / node console.  You'll get a parse error.  In order to lint, we can't have parse errors, so I changed this fenced code block in a way that should not affect its display while not calling it "javascript" (which it technically isn't).  I wrote briefly about this in the PR.